README.md

```diff
@@ -61,6 +61,10 @@ These scripts can all be run from the root level of the repo:
   * Runs JS unit tests
   * Lints JS using ESLint
   * Lints Sass using stylelint
+* `yarn test:e2e`
+  * Runs end to end tests
+* `yarn test:e2e packages/core/Autocomplete`
+  * Runs a single end to end test, Autocomplete is an example
```

Comment:
Really minor, but what do you think about 
`Runs a single end to end test, this example runs Autocomplete`?